bluesky/callbacks/mpl_plotting.py

```diff
@@ -200,7 +200,10 @@ def __init__(self, x, y, I, *, xlim=None, ylim=None,
         ax.margins(.1)
         self._xdata, self._ydata, self._Idata = [], [], []
         self._norm = mcolors.Normalize()
+        self._minx, self._maxx, self._miny, self._maxy = (None,)*4
```

Comment:
`self._minx = self._maxx = self._miny = self._maxy = None` is a slightly more boring way to do this.

Reply:
thanks for the suggestion!